packages/dd-trace/src/config.js

```diff
@@ -132,6 +132,11 @@ class Config {
       process.env.DD_TRACE_EXPERIMENTAL_INTERNAL_ERRORS_ENABLED,
       false
     )
+    const DD_APPSEC_ENABLED = coalesce(
+      options.appsec && options.appsec.beta,
+      process.env.DD_APPSEC_BETA,
```

Comment:
As mentioned above, this should be `DD_TRACE_EXPERIMENTAL_APPSEC_ENABLED` with the corresponding config.

Reply:
Damn, for the programmatic config that's fine, but for the env var we're trying to have the same name over all technologies, and that's the naming "they" decided. Do you have a strong opinion on this ?

Reply:
It's confusing for users if we're not consistent with how to enable disable features. I'm not saying splitting them out this way is wrong, but if we're going to do that it should be consistent across products within a language, which is more important than being consistent across languages.

Reply:
```js
    const DD_APPSEC_ENABLED = coalesce(
      options.experimental && options.experimental.appsec,
      process.env.DD_EXPERIMENTAL_APPSEC_ENABLED,
      false
    )
```
Is that ok with you ? (dropped the `TRACE`)